File: php/isPrime.php
<?php

$primes = 0;

function isPrime($number){
   for ($i = 2; $i < $number; $i++ )
       if ($number % $i == 0)
           return false;
   return true;
}

for ($k = 0; $k < 200000; $k++)
   if (isPrime($k))
       $primes++;
echo "\n".$primes."\n";

File: perl/isPrime.pl
$primes = 0;

sub isPrime{
   my $number = @_[0];
   for ($i = 2; $i < $number; $i++ ){
       my $rest = $number % $i;
       if ($rest eq 0){
           return 0;
       }
   }
   return 1;
}


for ($k = 2; $k < 200000; $k++){
    if (isPrime($k) eq 1){
       $primes++;

    }
}
print " $primes \n";
